problems/object-keys/problem.md

```diff
@@ -1,5 +1,44 @@
----
+Javascript provides a native way of listing all the available keys of an object. This can be helpful for looping through all the properties of an object and manipulating their values accordingly.
```

Comment:
Although it is a minor point, `Javascript` may be difficult for beginners to understand, so please describe `JavaScript` more accurately.

Reply:
I'm not sure that I'm following you here. Are you suggesting I replace `Javascript` with `JavaScript`?

Reply:
Yes 😄

Reply:
Done 👍